### PR TITLE
Feat/develop before third sprint review

### PR DESCRIPTION
**Pull request by dimitrijevic12**

### CODE REVIEW

File: PatientWebApplication/HealthClinic.DAL/Service/OperationService.cs [hunk 1]
```diff
@@ -197,23 +197,31 @@ private Boolean CheckIfOperation(String stringToCheck)
...
         {
             return GetOperationsForPatient(patientId).FindAll(operation => date == UtilityMethods.ParseDateInCorrectFormat(operation.Date));
         }
 
+        /// <summary> This method is creating a set out of list of doctor's and patient's operations on specific date. </summary>
+        /// <returns> <c>HashSet</c> of operations. </returns>
         public HashSet<Operation> CreateOperationtSetForDate(DateTime date, int doctorId, int patientId)
         {
             HashSet<Operation> appointmentsSet = new HashSet<Operation>(GetAllOperationsByDateAndDoctor(date, doctorId));
             appointmentsSet.UnionWith(GetAllOperationsByDateAndPatient(date, patientId));
```
Comment (velicko-tam) [inline PatientWebApplication/HealthClinic.DAL/Service/OperationService.cs:219]: Chnage return statement to return appointmentsSet.UnionWith(GetAllOperationsByDateAndPatient(date, patientId))

**Review by velicko-tam — CHANGES_REQUESTED** [review]
PR is not approved. Marked parts need to be changed.

**Review by trki293 — APPROVED** [review]
PR is approved. Good principles Clean code are followed.

**Review by velicko-tam — APPROVED** [review]
PR is approved.

**Pull request merged**